src/PlaylistController.js

```diff
@@ -0,0 +1,66 @@
+// import assert from 'assert';
+import GenericController from './GenericController';
+import Playlist from './Structs/Playlist';
+
+export default class PlaylistController extends GenericController {
+  constructor(...args) {
+    super(...args);
+    if (window.APPCONTEXT && window.APPCONTEXT.Go && window.APPCONTEXT.Go.h && window.APPCONTEXT.Go.h.length) {
+      this._playlists = Object.assign({}, window.APPCONTEXT.Go.h[0].Hi);
+      this._watchPlaylistObject();
+    } else {
+      this._playlists = {};
+      console.error('Could not listen to playlist changes in the expected location'); // eslint-disable-line
```

Comment:
Jokes, I totally remember why we need the `this_playlists` logic.  It's because google (at random points in time, mostly a few hours have passed) remove the playlists from the local cache.

By using `Object.assign({}, this._playlists, window.APPCONTEXT.Go.h[0].Hi)` we are saving playlists that may be removed from the cache later.